price-feeder/oracle/oracle.go

```diff
@@ -276,6 +276,7 @@ func GetComputedPrices(
 	providerPairs map[provider.Name][]types.CurrencyPair,
 	deviations map[string]sdk.Dec,
 ) (prices map[string]sdk.Dec, err error) {
+
```

Comment:
Is this a new lint rule?

Reply:
this file was not autoformatted with gofmt. I just opened it and saved it.